metadata

```diff
@@ -0,0 +1,7 @@
+[Application]
```

Comment:
I don't think we need this at all.

Reply:
When I remove the metadata file and the reference to it the flatpak won't build at all. 100% reproducible. Ideas?

Reply:
What error?

Reply:
Updated to a newer flatpak and this bug looks resolved.